frontend/src/Editor/Components/Timeline.jsx

```diff
@@ -15,7 +15,11 @@ export const Timeline = function Timeline({ height, darkMode, properties, styles
             <li key={index}>
               <div className="list-timeline-icon" style={{ backgroundColor: item.iconBackgroundColor }}></div>
               <div className="list-timeline-content">
-                {!hideDate && <div className={`list-timeline-time ${darkModeStyle}`}>{item.date}</div>}
+                {!hideDate && (
+                  <div className={`list-timeline-time ${darkModeStyle}`} style={{ top: 'auto' }}>
```

Comment:
Can you please change the style of the class ( `list-timeline-time` ) instead of inline css?

Reply:
Yes sure